libhb/encavcodec.c

```diff
@@ -228,6 +228,21 @@ int encavcodecInit( hb_work_object_t * w, hb_job_t * job )
                     break;
             }
         }break;
+        case AV_CODEC_ID_AV1:
+        {
+            switch (job->vcodec) {
+                case HB_VCODEC_SVT_AV1:
```

Comment:
This SVT-AV1 part is unnecessary.

Reply:
So this codec, is used now directly without FFmpeg?